### PR TITLE
Functioning Carousel

### PR DESCRIPTION
**Pull request by Daniel-Ghaly**
implemented functioning carousel

### CODE REVIEW

File: client/dist/stylesheet.css [hunk 1]
```diff
@@ -365,7 +413,7 @@ html {
   border: 1px solid black;
 }
 
-img {
+img .related {
```
Comment (FlightfulKiwi) [inline client/dist/stylesheet.css]: Change to `img.related-img` (no spaces). This is what we agreed to with Kavi. You can very easily make this fix here in Github.

File: client/src/components/Overview.jsx [hunk 2]
```diff
@@ -52,7 +52,7 @@ class Overview extends React.Component {
         this.setState({rating: rating});
       });
 
-    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/products/40344', {
+    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/products/40349', {
       headers: {
         authorization: token
       }
```
Comment (FlightfulKiwi) [inline client/src/components/Overview.jsx:58]: You do not have to implement this yet; we will revisit this later.
- Do not use the API urls anymore; use localhost to make GET requests
- Remove headers once you've made the change above; the header is handled in the Server

Also, right now you're hard-coding the product_id. I imagine you will change this later.

**Review by FlightfulKiwi — CHANGES_REQUESTED** [review]
Small change on CSS stylesheet requested. Let me know if you have questions about how to quickly make that change here in Github.

Other comment in Overview.jsx can be addressed as a group after Presentations.

File: client/dist/stylesheet.css (hunk 1 above)
Comment (Daniel-Ghaly) [reply client/dist/stylesheet.css]: change made

File: client/src/components/Overview.jsx (hunk 2 above)
Comment (Daniel-Ghaly) [reply client/src/components/Overview.jsx:58]: Will address this after. Thanks for the heads up!

**Review by FlightfulKiwi — APPROVED** [review]

**Pull request merged**